tdp_core/server/visyn_server.py

```diff
@@ -121,7 +121,8 @@ def create_visyn_server(
     # Load all namespace plugins as WSGIMiddleware plugins
     for p in router_plugins:
         _log.info(f"Registering router: {p.id}")
-        app.include_router(p.load().factory())
+        prefix = getattr(p, "namespace", "")  # consistent behaviour with flask plugins
+        app.include_router(p.load().factory(), prefix=prefix)
```

Comment:
Let's move away from using the registry for appending routers. 

Could you: 
1) Revert this change (just to keep it in-line with the existing behaviour --> we should look at that separately)
2) Add the `init_app(app: FastAPI)` callback to the VisynPlugin in the `__init__.py` and register the router there (via standard FastAPI machanism). 
That's just much clearer imo.